src/scss/skin-modern/components/_listbox.scss

```diff
@@ -9,10 +9,12 @@
 
     display: block;
     font-size: .8em;
-    height: 100%;
     min-width: 10em;
     padding: .5em;
-    width: 100%;
+
+    &::before {
+      display: none;
```

Comment:
Why is this necessary?

Reply:
Because the `::before` is blocking content. For buttons without images this would mean that the text is no longer centered.